shared/utility/skill/WalkGenerator.hpp

```diff
@@ -90,6 +90,9 @@ namespace utility::skill {
 
             /// @brief Option to only switch between planted foot if the next foot is planted
             bool only_switch_when_planted = false;
+
+            /// @brief Option to use balance control when walking
+            bool use_balance_control = false;
```

Comment:
This is never used for walk generator stuff, so I think it should be moved to a variable on the walk/kick class